src/conf_mode/nat.py

```diff
@@ -199,7 +211,7 @@ def verify(nat):
                 Warning(f'rule "{rule}" interface "{config["outbound_interface"]}" does not exist on this system')
 
             if not dict_search('translation.address', config) and not dict_search('translation.port', config):
-                if 'exclude' not in config:
+                if 'exclude' not in config and 'member' not in config['balance']:
```

Comment:
What happens if `balance` is not specified? A KeyError will be reported.

Maybe use: `if 'exclude' not in config and dict_search('balance.member', config) == None:`

Reply:
`balance` is always defined, since in `hash` section we have a default value. So if firewall is defined, we always get the default value `balance hash random` (unless something different is defined).. I faced with that issue while preparing this PR